src/components/molecules/ArticleTrashTable/index.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <table class="article-trashed-table">
+    <thead class="article-trashed-table__head">
+      <tr>
+        <th v-for="(thead, index) in theads" :key="index">
+          <app-text tag="span" theme-color bold>
+            {{ thead }}
+          </app-text>
+        </th>
+      </tr>
+    </thead>
+    <tbody class="article-trashed-table__body">
+      <tr v-for="article in targetArray" :key="article.id">
+        <td>
+          <app-text tag="span" small>
+            {{ truncate(article.title,30) }}
```

Comment:
[nits]
`,`の後ろは半角開けましょう！

```suggestion
            {{ truncate(article.title, 30) }}
```

Reply:
1c9692598f78fe9fa8e9b7a54f04794a15fdabbd
,の後ろ半角空白挿入いたしました！